server.js

```diff
@@ -24,9 +17,116 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+const Animal = mongoose.model("Animal", {
+  name: String,
+  scientificName: String,
+  doesItFart: Boolean,
+  doesItMaybeFart: Boolean,
+  descriptionExists: Boolean,
+  descriptionNotes: String,
+  paperLink: String,
+  verifiedBy: String,
+});
+
+// if (process.env.RESET_DB) {
```

Comment:
Im unsure if we should have this part in production or not - but Im glad you kept it even if commented out as shows that you have understood the meaning and structure. I noticed that some of the deployed projects takes time to load in browser - including mine. Wonder it this might have something to do with it as your page is loading very fast.